### PR TITLE
Adds the createError module for use in Telescope

### PR DESCRIPTION
**Pull request by Metropass**
As per the documentation of create-error, the use of the module goes as follows:

```js
createError(value, msg)
```
`value`: A value that the error would contain, for example if we would want to store a 404 error, the value would be '404'.  This argument is not tied to a specific type, so you're allowed to pass in types that aren't an `int`. 

`msg`: A message detailing what the error is about, it can be anything, so long as it's a string.

Accessing Members:
If you wish to access `value` and or `msg`, you can do so like this:
```js
var example_error = createError(200, "OK")
```
Accessing Value - `example_error.value`
Accessing Message - `example_error.message`

### CODE REVIEW

**Review by humphd — CHANGES_REQUESTED** [review]
You need to rebase and do this off of `master`

**Comment by Metropass** [comment]
> You need to rebase and do this off of `master`

Everything should be rebased now, I just gotta add the module on test.js, and then I'll squash

File: package.json [hunk 1]
```diff
@@ -17,6 +17,7 @@
     "@elastic/ecs-pino-format": "^1.0.0",
     "@godaddy/terminus": "^4.6.0",
     "cors": "2.8.5",
+    "create-error": "^0.3.1",
```
Comment (humphd) [inline package.json:20]: Use the `http-errors` module (already installed below).  I might have mislead you on which one to use, sorry about that.  They work the same way (e.g., both have `createError`), see https://www.npmjs.com/package/http-errors.

File: src/createError.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+const create_error = require("create-error");
```
Comment (humphd) [inline src/createError.js:1]: This isn't Python.  In JS you use lowerCamelCase not snake_case: `createError`

**Review by humphd — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (Metropass) [reply package.json:20]: Will do! Thanks for the heads up

File: test.js [hunk 3]
```diff
@@ -351,6 +352,15 @@ describe("Satellite()", () => {
       expect(res.ok).toBe(false);
       expect(res.status).toEqual(401);
 
+      // Protected should work with correct authorization token
```
Comment (humphd) [inline test.js:355]: Why this change?

File: src/createError.js [hunk 4]
```diff
@@ -1,9 +1,8 @@
-const create_error = require("create-error");
+const createHttpError = require("http-errors");
 
 function createError(value, msg) {
-  const e = create_error("MainError", { message: [] });
-  const v = new e(msg, { value: value });
-  return v;
+  const err = createHttpError(value, msg);
```
[the remark this answers is not in the document]
Comment (humphd) [reply src/createError.js:4]: Delete this file

**Review by humphd — CHANGES_REQUESTED** [review]
A couple of things to do, and also please add docs similar to how the logger is documented, to show people how to use this:

https://github.com/Seneca-CDOT/satellite#logger

File: test.js (hunk 3 above)
Comment (Metropass) [reply test.js:355]: Oh dear, I think that was due to my editor, let me clean that up

**Comment by humphd** [comment]
This still needs docs as I mentioned above.

**Comment by humphd** [comment]
Also, see how almost every line of test.js got changed due to prettier?  Next time it would have been a good idea to run prettier on the code when you put in the prettier config, so it matches the new config.

**Comment by Metropass** [comment]
> This still needs docs as I mentioned above.

I'll add the docs right now

**Comment by Metropass** [comment]
> Also, see how almost every line of test.js got changed due to prettier? Next time it would have been a good idea to run prettier on the code when you put in the prettier config, so it matches the new config.

Yeah, I noticed that prettier changed all the lines, I should've done that during the Prettier patch. 
I'll know that for next time.

**Review by humphd — APPROVED** [review]
